statsmodels/discrete/discrete_model.py

```diff
@@ -1571,6 +1597,32 @@ def score(self, params):
         else:
             return score
 
+    def score_factor(self, params):
+        if self._transparams:
+            alpha = np.exp(params[-1])
+        else:
+            alpha = params[-1]
+
+        params = params[:-1]
+        p = self.parameterization
+        exog = self.exog
+        y = self.endog[:,None]
+        mu = self.predict(params)[:,None]
+        mu_p = np.power(mu, p)
+        a1 = 1 + alpha * mu_p
+        a2 = mu + alpha * mu_p * y
+        a3 = alpha * p * mu ** (p - 1)
+        a4 = a3 * y
```

Comment:
do any of `a1,a2,a3,a4` have meaningful interpretations?  (e.g. `dmudb` below im assuming means `\frac{\partial\mu}{\partial b}` in which case its a nice informative name)